From 11024c9a5a44b84156edc31b8f2876d88db10893 Mon Sep 17 00:00:00 2001
From: Clemens Lang <neverpanic@gmail.com>
Date: Sun, 24 Jun 2018 16:19:08 +0200
Subject: [PATCH] Revert "Build with MacOSX10.11 SDK"

This reverts commit e28713286ee2cc63a1c619ee67dadb7ca30f0d5e.

The path to the SDK is hardcoded, which is not very useful. Revert this
commit to fix the build.

Upstream-Status: Inappropriate [Configuration]
---
 target | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git ./target ./target
index 2620c6a7..6589cdd9 100644
--- ./target
+++ ./target
@@ -1,9 +1,7 @@
 PLIST_FLAGS = -d'APP_NAME=$APP_NAME' -d'APP_VERSION=$APP_VERSION' -d'APP_MIN_OS=$APP_MIN_OS'
 
-SDK = /Users/duff/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk
-
 FLAGS += -c -pipe -fPIC -gdwarf-2
-FLAGS += -m64 -mmacosx-version-min=$APP_MIN_OS -isysroot "$SDK"
+FLAGS += -m64 -mmacosx-version-min=$APP_MIN_OS
 FLAGS += -funsigned-char
 FLAGS += -D'NULL_STR="\uFFFF"'
 FLAGS += -DREST_API='"$rest_api"'
@@ -21,7 +19,7 @@ OBJCXX_FLAGS += -fvisibility=hidden -std=c++14 -fobjc-abi-version=3
 OBJCXX_FLAGS += -fobjc-arc -fobjc-call-cxx-cdtors
 LIBS         += c++
 
-LN_FLAGS += -m64 -mmacosx-version-min=$APP_MIN_OS -isysroot "$SDK"
+LN_FLAGS += -m64 -mmacosx-version-min=$APP_MIN_OS
 LN_FLAGS += -fvisibility=hidden
 LN_FLAGS += -Wl,-dead_strip
 LN_FLAGS += -Wl,-dead_strip_dylibs
-- 
2.18.0

